process-migration/image.yaml

```diff
@@ -0,0 +1,64 @@
+schema_version: 1
+
+name: "rhpam-7/rhpam75-process-migration-openshift"
+description: "Red Hat Process Automation Manager Process Migration 7.5 container image"
+version: "1.0"
+from: "ubi8:8-released"
+labels:
+    - name: "com.redhat.component"
+      value: "rhpam-7-rhpam75-process-migration-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Platform for running Red Hat Process Automation Manager Process Migration"
+    - name: "io.k8s.display-name"
+      value: "Red Hat Process Automation Manager Process Migration 7.5"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "javaee,rhpam,rhpam7"
+envs:
+    - name: "SCRIPT_DEBUG"
+      example: "true"
+      description: "If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed."
+    - name: JBOSS_KIE_EXTRA_CLASSPATH
+      example: "mariadb-java-client-2.4.2.jar,postgresql-42.2.6.jar"
+      description: "Comma separated list of libraries to add to the classpath. They must be accessible."
+    - name: JBOSS_KIE_EXTRA_CONFIG
+      example: "/opt/rhpam-process-migration/config-extra/pim-config.yaml"
+      description: "Path to a file including a yaml configuration file to override default Thorntail configuration and target KIE Servers"
+ports:
+    - value: 8080
+modules:
+      repositories:
+          - name: cct_module
+            git:
+                  url: https://github.com/jboss-openshift/cct_module.git
+                  ref: master
+          - name: rhpam-7-image
+            git:
+                  url: https://github.com/jboss-container-images/rhpam-7-image.git
+                  ref: master
+          - name: jboss-kie-modules
+            git:
+                  url: https://github.com/jboss-container-images/jboss-kie-modules.git
+                  ref: master
+      install:
+          - name: jboss.container.openjdk.jdk
+            version: "11"
+          - name: dynamic-resources
+          - name: rhpam-7-process-migration
+          - name: os-logging
+          - name: jboss-kie-process-migration
+packages:
+      content_sets_file: content_sets.yml
+      install:
+          - hostname
+osbs:
+      configuration:
+          container_file: container.yaml
+      repository:
+          name: containers/rhpam-7-process-migration-openshift
```

Comment:
make sure you talk to Ivo Bek so he can request the container/branch for you in osbs asap.